instant-xml/tests/all.rs

```diff
@@ -81,3 +91,104 @@ fn struct_with_custom_field_wrong_prefix() {
     .to_xml()
     .unwrap();
 }
+
+#[test]
+fn struct_with_custom_field_from_xml() {
+    assert_eq!(
+        StructWithCustomFieldFromXml::from_xml("<StructWithCustomFieldFromXml flag_attribute=\"true\" xmlns=\"URI\" xmlns:bar=\"BAZ\" xmlns:foo=\"BAR\"><bar:flag>false</bar:flag><Nested><flag>true</flag></Nested></StructWithCustomFieldFromXml>").unwrap(),
```

Comment:
BTW, I think I missed this on the serializer side, but this is actually wrong due to namespaces.

Because the XML here sets the default namespace to `URI`, the `Nested` tag must be interpreted as `{URI}baz`, so deserialization should fail in this case because the `Nested` type does not declare its namespace as `URI`.

Reply:
This along with other identified issues, will be addressed in this PR: #10